src/Microsoft.Azure.SignalR.AspNet/Infrastructure/ServiceConnection.cs

```diff
@@ -90,7 +91,15 @@ protected override Task OnMessageAsync(ConnectionDataMessage connectionDataMessa
                 {
                     var payload = connectionDataMessage.Payload;
```

Comment:
This code needs to be changed to parse text the text message separator.

Reply:
already parsed when dispatching message from the runtime side.